### DISCUSSION TITLE
TMUX

### DISCUSSION THREAD

**Discussion by TheCollinsByte** [2KAbhishek/dots2k#15] [Q&A]
synchronize-panes toggle:
prefix + Ctrl-s is used by tmux resurrect

**Comment by TheCollinsByte** [comment]
Correct Me if I am wrong

**Comment by 2KAbhishek** [comment]
I'm not aware if synchronize panes has a keyboard shortcut, usual way to trigger it is: `prefix :setw synchronize-panes`
